library/IvozProvider/Klear/Filter/QueueStrategies.php

```diff
@@ -1,18 +1,23 @@
 <?php
 
-use \IvozProvider\Mapper\Sql as Mapper;
+use Ivoz\Ast\Domain\Model\Queue\Queue;
```

Comment:
I think it should be better to work with `Ivoz\Provider\Domain\Model\Queue\Queue` here.

The other entity is used to preload asterisk data only.

The below $_queue class attribute is Provider QueueDTO too